code-challenges.js

```diff
@@ -9,67 +9,156 @@
 // Use the test variables provided below. Expected output: "HyThrLrnStdnt" "LvJvScrpt"
 
 var stringWithVowels1 = "HeyThereLearnStudent"
 var stringWithVowels2 = "ILoveJavaScript"
 
-
-
-
+const removedVowels = (array) => {
+// // 	// Create variable to store the letters
+    let letters = array.split(""); 
+    let letters2 =letters.filter(value => {
+        if (! ["a", "e", "i", "o", "u"].includes(value)) {
+            return true
+        } else {
+            return false
+        }
+    })
+    return letters2.join('')
+}
+console.log(removedVowels(removedVowels(stringWithVowels2)))
+// //         // create an argument that check to see if statements are true
+//             // if true, then it will push whatever does not match the vowels provided
+              
+// // Expected output "HyThrLrnStdnt" "LvJvScrpt"
 
 // --------------------5) Copy/paste your code from #4. Refactor your code to include non-string edge cases. Inform your user that the variable passed into the vowel removal function is not a string.
 // Use the test variables provided below. Expected output: "true is not a string" "42 is not a string" "mCdngMstr"
 
-var refactorTester1 = true
-var refactorTester2 = 42
-var refactorTester3 = "IAmACodingMaster"
-
-
+// var refactorTester1 = true
+// var refactorTester2 = 42
+// var refactorTester3 = "IAmACodingMaster"
+// // pasted the previous syntax from questions 4.
+// // need function to include nonstreing edges
+// const removedVowels = (array) => {
+//         let letters = array.split(""); 
+//         let letters2 =letters.filter(value => {
+//             // need it to check for uppercase and lowercase letters
+//             if (! ["a", "e", "i", "o", "u"].includes(value)) {
+//                 // need it to check nonstring edges
+//                 return true
+//             } else {
+//                 return false
+//             }
+//         })
+        
+//  if (typeof value === "number" || typeof value === "boolean" {
+//     return `${value} is not a string`
+//     return letters2.join('')
+// }
+//     }
+//     console.log(removedVowels(refactorTester2))
+    // can't get it to read the uppercase letters too. i know i have to put .toLowerCase
+    // but i can't find anywhere to put it
 
 
 
 // --------------------6) Create a function that takes in an array of strings and returns the string with the most characters.
 // Expected output: "chopsticks" "stemware"
 
-var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
-var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
-
+// var utensils = ["fork", "knife", "tongs", "chopsticks", "skewer"]
+// var vessels = ["cup", "mug", "stemware", "glass", "tumbler", "tea cup"]
 
+// const highestNumWord = (array, index) => {
 
+// for (let i = 1; i < array.length; i++) {
+//     // Check if the next element in the array has more letters
+//     if (array[i] > highestNumWord) {
```

Comment:
this seems to be referencing the function itself. Try creating instructions for what you would need to do to determine whether the first value in the array is the longest word...then try doing the same instructions for the next value in the array.